fuzzers/eclipser/fuzzer.py

```diff
@@ -94,7 +94,8 @@ def eclipser(input_corpus, output_corpus, target_binary):
     if os.listdir(input_corpus):  # Specify inputs only if any seed exists.
         command += ['-i', input_corpus]
     print('[eclipser] Run Eclipser with command: ' + ' '.join(command))
-    subprocess.Popen(command)
+    with subprocess.Popen(command):
+        pass
```

Comment:
Not sure if this is the best solution:
`pylint` complains about using `Popen` without using `with`.
This requirement (kind of) makes sense when `Popen` writes to a file descriptor, as `with` ensures that the [writing is done properly](https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/consider-using-with.html).
But I am not sure if that applies here, as this `Popen` did not specify any file descriptor and the default value is `None`.
Would you prefer me to ignore that error instead?

Reply:
There are several other fixes for this error: I grouped them in one commit to make it easier to revert them.

Reply:
Can you disable this warning just for Popen? If you can let's just do this in .pylintrc (this is kind of burdensome for files in fuzzers/)

Reply:
I reckon that would require `pylint` to ignore a specific regex (e.g., `.*Popen.*`) for a specific error (e.g., `consider-using-with`).
I did not find a way to do so.
Alternatively, we can either disable `consider-using-with` errors (which will also ignore incorrect `open`), or ignore all `fuzzers/*/fuzzer.py` files (which will ignore all other errors in those files. Neither is idea.

Reply:
I will keep the current way (using `with`), but please let me know if you'd prefer me to ignore each of them manually (`# pylint: disable:consider-using-with`) or there is a better solution : )

Reply:
+1